app/components/pagination-item.tsx

```diff
@@ -0,0 +1,101 @@
+import { Job } from "~/routes/career";
+import { Button } from "./ui/button";
+import { ChevronLeft, ChevronRight, MoreHorizontal } from "lucide-react";
```

Comment:
This pagination logic is not correct, please use ShadCN pagination component

Reply:
someone implemented the pagination already https://github.com/hngprojects/hng_boilerplate_remix/pull/214

base on my  task description i thought it is my responsibility